src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -0,0 +1,386 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+use Omnipay\Common\Message\ResponseInterface;
+
+class ApplePayAuthorizeRequest extends \Omnipay\Common\Message\AbstractRequest
```

Comment:
can you add a classblock explaining how to use this? see the other requests for examples